src/Reporter.js

```diff
@@ -44,6 +44,7 @@ class Reporter {
         this.includeAllCasesDuringCreation = configService.includeAllCasesDuringCreation();
         this.includeAllFailedScreenshots = configService.includeAllFailedScreenshots();
         this.ignorePendingTests = configService.ignorePendingCypressTests();
+        this.ignoreMissingCaseIDs = configService.includeCasesOnlyIfExistInRun();
```

Comment:
change the config to match
```suggestion
        this.ignoreMissingCaseIDs = configService.ignoreMissingCaseIds();
```